base/environment.yml

```diff
@@ -2,5 +2,7 @@ channels:
 - conda-forge
 dependencies:
 - ipywidgets
-- pyiron_base =0.5.7
-- seaborn =0.11.2
+- pyiron_base =0.5.31
```

Comment:
Can we already use `pyiron_base=0.5.32` that should fix the SQLalchemy issues? Or we temporarily enforce `sqlalchemy<2.0.0`. https://github.com/pyiron/pyiron_atomistics/pull/949

Reply:
Yeah, sure. I edited this file before the sqlalchemy issue arised and forgot to check again... My bad, sorry.

Reply:
If you also need `pyiron_atomistics` then please add `sqlalchemy<2.0.0`. `pyiron_base` is currently backwards compatible https://github.com/conda-forge/pyiron_base-feedstock/pull/147

Reply:
I'll do that 👍